tests/test_airflow.py

```diff
@@ -1,193 +0,0 @@
-import pytest
```

Comment:
@smpiano I don't remember, did you  port the test to the new gfw-airflow repo?

Reply:
@enriquetuya yes, I moved the test where the airflow operators are. I mean, the new `airflow-gfw` repo. Here is the [test_airflow.py](https://github.com/GlobalFishingWatch/airflow-gfw/pull/1/files#diff-a55eeb5db2d79fedfb6b8b772a475204)